submit.py

```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env python
-import ConfigParser
+from __future__ import print_function
 import optparse
 import os
 import sys
 import itertools
-import mimetools
 import mimetypes
-import urllib
-import urllib2
 import traceback
-
+import random
+import string
+if sys.version_info[0] >= 3:
+    _PYTHON_VERSION = 3
+    import configparser
+    import urllib.parse
+    import urllib.request
+    import urllib.error
+else:
+    # Python 2, import modules with Python 3 names
+    _PYTHON_VERSION = 2
+    import ConfigParser as configparser
+    import urllib
+    import urllib2
+    urllib.request = urllib.error = urllib2
```

Comment:
Usually I wouldn't suggest this, but maybe it would look a little cleaner to import the functions/classes needed rather than patching the `urllib` module, like:
```
from urllib.request import Request # py3
...
from urllib2 import Request # py2
```
and similar for `error` and `parse`
or does that look hackier still?

Or there is 

```
from urllib import parse, request, error
...
import urllib as parse
import urllib2 as request
import urllib2 as error
```